modules/social_features/social_event/config/optional/message.template.activity_on_events_im_organizing.yml

```
@@ -0,0 +1,36 @@
+langcode: en
+status: true
+dependencies:
+  module:
+  - activity_logger
+third_party_settings:
+  activity_logger:
+    activity_bundle_entities:
+      event_enrollment-event_enrollment: event_enrollment-event_enrollment
+    activity_action: create_entitiy_action
+    activity_context: organizer_activity_context
+    activity_destinations:
+      notifications: notifications
+      email: email
+    activity_create_direct: 1
+    activity_aggregate: 0
+    activity_entity_condition: ''
+template: activity_on_events_im_organizing
+label: 'Activity on events I am organizing'
+description: 'Activity on events I am organizing'
+text:
+  -
+    value: '<p><a href="[social_event:enrolled_event_url]">[social_event:enrolled_user]</a> has enrolled to [social_event:event_iam_organizing] you are organizing</p>'
```

Previously we were changing the messages for group to add "the group" in front of the group name. I think we should do the same here.

```suggestion
    value: '<p><a href="[social_event:enrolled_event_url]">[social_event:enrolled_user]</a> has enrolled to the event [social_event:event_iam_organizing] you are organizing</p>'
```